client/src/app/gui/rtcp_packets_table.rs

```diff
@@ -0,0 +1,251 @@
+            ("Time", "Packet arrival timestamp"),
+            ("Source", "Source IP address and port"),
+            ("Destination", "Destination IP address and port"),
+            ("Type", "Type of the RTCP packet"),
+            ("Data", "Data specific to RTCP packet's type"),
+        ];
+
+        TableBuilder::new(ui)
+            .striped(true)
+            .resizable(true)
+            .stick_to_bottom(true)
+            .column(Column::initial(60.0).at_least(60.0))
+            .column(Column::initial(70.0).at_least(70.0))
+            .columns(Column::initial(150.0).at_least(150.0), 2)
+            .column(Column::initial(170.0).at_least(170.0))
+            .column(Column::remainder())
+            .header(30.0, |mut header| {
+                for (label, desc) in header_labels {
+                    header.col(|ui| {
+                        ui.heading(label.to_string())
+                            .on_hover_text(desc.to_string());
+                    });
+                }
+            })
+            .body(|body| {
+                self.build_table_body(body);
+            });
+    }
+
+    fn build_table_body(&mut self, body: TableBody) {
+        let streams = &self.streams.borrow();
+        let rtcp_packets: Vec<_> = streams
+            .packets
+            .values()
+            .filter_map(|packet| {
+                let rtcp = match packet.contents {
+                    SessionPacket::Rtcp(ref rtcp) => rtcp,
+                    _ => return None,
+                };
+
+                Some(rtcp.iter().map(|pack| (packet.id, pack)))
+            })
+            .flatten()
+            .collect();
+
+        if rtcp_packets.is_empty() {
+            return;
+        }
+
+        let heights = rtcp_packets.iter().map(|(_, pack)| get_row_height(pack));
+
+        let mut last_id = 0;
+        let mut next_ix = 1;
+
+        let first_ts = streams.packets.get(0).unwrap().timestamp;
+        body.heterogeneous_rows(heights, |ix, mut row| {
+            let (id, rtcp) = rtcp_packets.get(ix).unwrap();
+            let packet = streams.packets.get(*id).unwrap();
+
+            if *id != last_id {
+                log::info!("DUPA {}, {}", *id, last_id);
+                last_id = *id;
+                next_ix = 1;
+            }
+
+            row.col(|ui| {
+                ui.label(format!("{} ({})", id, next_ix));
+                next_ix += 1;
+            });
+            row.col(|ui| {
+                let timestamp = packet.timestamp - first_ts;
+                ui.label(format!("{:.4}", timestamp.as_secs_f64()));
+            });
+            row.col(|ui| {
+                ui.label(packet.source_addr.to_string());
+            });
+            row.col(|ui| {
+                ui.label(packet.destination_addr.to_string());
+            });
+            row.col(|ui| {
+                ui.label(rtcp.get_type_name().to_string());
```

Comment:
```suggestion
                ui.label(RichText::from(rtcp.get_type_name()).strong());
```

Reply:
Tried it and honestly, I don't think it looks better. Gonna leave as it is, we can always change it later.